google-junk/googlef7300098ace1e54a.html

```diff
@@ -0,0 +1 @@
+google-site-verification: googlef7300098ace1e54a.html
```

Comment:
This should remain in the root directory.

Reply:
okay